### DISCUSSION TITLE
[SOLVED] Theme not working properly on MacOS

### DISCUSSION THREAD

**Discussion by ari-gutierrez** [janoamaral/tokyo-night-tmux#84] [Ideas]
I've found myself trying to make the plugin work in MacOS Sonoma, but it was not clear for me the root cause, provided I'm running ZSH as my shell, and installed a newer version of bash via homebrew.

The problem was in how the PATH was built and how the `/usr/bin/env` interprets the `bash` existence into PATH; being the first one that appears which takes the precedence. This could be checked watching the PATH contents `$ echo $PATH` and testing the env precedence executing `$ /usr/bin/env bash`.

Probably this is a novice error, but it will be nice to notice it; or to eliminate the hard bash requirement, because it's mainly used to see the actual directory.

Cheers,
Ari

**Comment by damienbutt** [comment]
What is the solution to this? What did you do to solve the problem?

I have this issue and cannot get it to function correctly

![Screenshot by Dropbox Capture](https://github.com/janoamaral/tokyo-night-tmux/assets/22627489/27e618d4-c176-4041-bb22-5aeedeefeac6)

**Comment by janoamaral** [comment]
Hi, it's a matter of bash version. Check the requirements please.

**Reply by damienbutt** [reply]
Hey,

Thanks for coming back to me.

```sh
bash --version

GNU bash, version 5.2.26(1)-release (x86_64-apple-darwin23.2.0)
Copyright (C) 2022 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
```

I have set up my Mac so if I use bash it uses the Homebrew version above. 

```sh
which bash

/usr/local/bin/bash
```

Is this version sufficient? Or is it possible that the plugin is still somehow using the older Apple bundled bash? Is there a way I can force the plugin to use bash from `/usr/local/bin/bash`?

Thanks,
D

**Comment by ari-gutierrez** [comment]
As I've said, the problem was about the BASH version; but in my case it was because of the way the PATH was declared, and the macos default version took precedence over the homebrew installed one. It was just a point to improve a bit the documentation. Just that.
